src/misc/lv_timer.c

```diff
@@ -258,7 +258,7 @@ void _lv_timer_core_deinit(void)
     _lv_ll_clear(timer_ll_p);
 }
 
```

Comment:
just a question: why is it changed to uint32, if it only contains percentage, and even the [idle_last](https://github.com/lvgl/lvgl/blob/3e819741a56971c3e2065dcb603f3702d4b9c2ff/src/misc/lv_timer.h#L63) is uint8 ?

Reply:
Because in v9 I've tried to have `uint32_t` or `int32_t` on APIs where possible to unify things a little bit.

Reply:
I thought it’s due to some performance “magic” in some MCUs that I’m not aware :-)